src/streamlink/plugins/wwenetwork.py

```diff
@@ -146,7 +114,8 @@ def _get_streams(self):
         except ValueError:
             start_point = 0
 
-        content_id = self._get_video_id()
+        video_id = self.match.group("video_id")
+        content_id = self._get_video_id(video_id)
```

Comment:
`content_id` is now always a truthy value, so the following if-statement is redundant. But as said, not really important.